identity/app/actions/AuthenticatedActions.scala

```diff
@@ -68,105 +64,117 @@ class AuthenticatedActions(
-    new AuthenticatedBuilder(authService.authenticatedUserFor, anyContentParser, unAuthorizedCallback)
-
-  def apiVerifiedUserRefiner: ActionRefiner[AuthRequest, AuthRequest] = new ActionRefiner[AuthRequest, AuthRequest] {
-    override val executionContext = ec
-
-    def refine[A](request: AuthRequest[A]) =
-      identityApiClient.me(request.user.auth).map {
-        _.fold(
-          errors => {
-            logger.warn(s"Failed to look up logged-in user: $errors")
-            Left(sendUserToSignin(request))
-          },
-          userDO => {
-            logger.trace("user is logged in")
-            Right(new AuthRequest(request.user.copy(user = userDO), request))
-          }
-        )
-      }
-  }
+  private def retrieveUserFromIdapiRefiner: ActionRefiner[AuthRequest, AuthRequest] =
+    new ActionRefiner[AuthRequest, AuthRequest] {
+      override val executionContext = ec
+
+      def refine[A](request: AuthRequest[A]) =
+        identityApiClient.me(request.user.auth).map {
+          _.fold(
+            errors => {
+              logger.warn(s"Failed to look up logged-in user: $errors")
+              Left(sendUserToSignin(request))
+            },
+            userDO => {
+              logger.trace("user is logged in")
+              Right(new AuthRequest(request.user.copy(user = userDO), request))
+            }
+          )
+        }
+    }
 
-  def apiUserShouldRepermissionRefiner: ActionRefiner[AuthRequest, AuthRequest] = new ActionRefiner[AuthRequest, AuthRequest] {
-    override val executionContext = ec
-
-    def refine[A](request: AuthRequest[A]) =
-      if (IdentityPointToConsentJourneyPage.isSwitchedOn && IdentityAllowAccessToGdprJourneyPageSwitch.isSwitchedOn)
-        decideConsentJourney(request)
-      else
-        Future.successful(Right(request))
-
-    private def decideConsentJourney[A](request: AuthRequest[A]) =
-      userEmailValidated(request) -> userHasRepermissioned(request) match {
-        case (false, _) => Future.successful(Left(sendUserToValidateEmail(request)))
-        case (true, false) => Future.successful(Left(sendUserToAllConsentsJourney(request)))
-        case (true, true) =>
-          newsletterService.subscriptions(request.user.getId, idRequestParser(request).trackingData).map { emailFilledForm =>
-            if (newsletterService.getV1EmailSubscriptions(emailFilledForm).isEmpty)
-              Right(request)
-            else
-              Left(sendUserToNewslettersConsentsJourney(request))
-          }
-      }
-
-    private def userHasRepermissioned(request: AuthRequest[_]): Boolean =
-      request.user.statusFields.hasRepermissioned.contains(true)
-
-    private def userEmailValidated(request: AuthRequest[_]): Boolean =
-      request.user.statusFields.isUserEmailValidated
-}
+  private def apiUserShouldRepermissionRefiner: ActionRefiner[AuthRequest, AuthRequest] =
```

Comment:
I'm wondering whether this `ActionRefiner` could instead be a slightly simpler `ActionFilter` because we seem to just be doing `Right(request)` without transforming the request. Might also apply to some of our other refiners, haven't checked.